units.go

```diff
@@ -85,10 +89,14 @@ func CharName(src string) string {
 		return "CT-5555 \"Fives\""
 	case "rex", "zrex":
 		return "CT-7567 \"Rex\""
+	case "Malak":
```

Comment:
Since we lowercase before comparing in the switch/case, we need to check always to lowercase too.

Reply:
fixed